packages/frontend/src/styles.css

```
@@ -344,3 +344,144 @@ h1 {
+}
+
+.log-panel__row {
+  display: grid;
+  grid-template-columns: max-content max-content auto 1fr;
+  align-items: baseline;
+  column-gap: 0.55rem;
+  row-gap: 0.25rem;
+  padding: 0.4rem 0.5rem;
+  border-radius: 6px;
+  background: rgba(22, 33, 31, 0.03);
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)